uWeather/app/src/main/java/dit257/mandalore/uweather/Probability.kt

```diff
@@ -1,36 +1,36 @@
 package dit257.mandalore.uweather
 
+import java.lang.Math.*
 import java.util.*
+import kotlin.math.abs
 
 class Probability {
 
     //Calculate probability based on difference from median
-    fun calcProbabilityMedian(all_last_temps:IntArray): Double {
```

Comment:
does this only work for the last Temperatures and not for future temperatures?

Reply:
This only works for the current temperature, a.k.a the latest temperature provided by the API request

Reply:
Resolve the conversations when you have understood :)

Reply:
I understand and agree